packages/crayons-core/src/components/country-phone/country-phone.scss

```diff
@@ -0,0 +1,15 @@
+/**
+  @prop --fw-select-width: Select width
```

Comment:
This will appear in documentation. Kindly explain.

Reply:
@Vignesh-Manogar-E3433 : Added in read.me (## CSS Custom Properties) How accord-in component added

Reply:
![Screenshot 2022-07-14 at 11 22 46 AM](https://user-images.githubusercontent.com/94836700/178909809-8209fe99-4a3a-48c4-ac61-921d74566806.png)

Reply:
This is an autogenerated section. Next time we generate build, this will get overwritten with whats in comments.

![Screenshot 2022-07-14 at 11 38 36 AM](https://user-images.githubusercontent.com/61269786/178912073-c3998ce5-7f9b-4821-ae23-663a0e6add24.png)

This is a screenshot when I build documentation.

Reply:
Fixed ✅